conans/model/settings.py

```diff
@@ -233,7 +233,10 @@ def copy_values(self):
 
     @staticmethod
     def loads(text):
-        return Settings(yaml.safe_load(text) or {})
+        try:
+            return Settings(yaml.safe_load(text) or {})
+        except yaml.YAMLError as ye:
+            raise ConanException("settings:invalid YAML format:%s" % str(ye))
```

Comment:
```suggestion
            raise ConanException("Invalid settings.yml format:%s" % str(ye))
```

Reply:
Just an extra space to the previous suggesting

```suggestion
            raise ConanException("Invalid settings.yml format: %s" % str(ye))
```